.travis.yml

```diff
@@ -55,6 +55,7 @@ matrix:
         # Check compatibility with the current astropy LTS release
         - python: 2.7
           env: ASTROPY_VERSION=LTS
+               SETUP_CMD='test --coverage'
```

Comment:
More to have a coverage of the LTS branch, otherwise either the coverage decreases significantly or I had to put a `# pragma no cover` after each of the arithmetic function.